controllers/todo.js

```diff
@@ -14,3 +14,21 @@ module.exports.save = function save(req, res){
     res.json({ id: data._id })
   })
 }
+
+module.exports.modify = function modify(req, res) {
+  const modified = new model(req.body)
```

Comment:
you dont need to create a model to use req.body, req.body is indeed a js object so, you can use it  like.. req.body.id and req.body.title instead of create the model (modified.id, modified.title)